lib/node_modules/@stdlib/stats/base/dists/logistic/mgf/benchmark/benchmark.js

```diff
@@ -21,28 +21,36 @@
 // MODULES //
 
 var bench = require( '@stdlib/bench' );
-var randu = require( '@stdlib/random/base/randu' );
+var Float64Array = require( '@stdlib/array/float64' );
+var uniform = require( '@stdlib/random/base/uniform' );
 var isnan = require( '@stdlib/math/base/assert/is-nan' );
-var EPS = require( '@stdlib/constants/float64/eps' );
 var pkg = require( './../package.json' ).name;
 var mgf = require( './../lib' );
 
 
 // MAIN //
 
 bench( pkg, function benchmark( b ) {
+	var len;
 	var mu;
 	var s;
 	var t;
 	var y;
 	var i;
 
+	len = 100;
+	t = new Float64Array( len );
+	mu = new Float64Array( len );
+	s = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		s[i] = uniform( 0.1, 0.5 );
+		t[i] = uniform( 0.0, 0.9 );
+		mu[i] = uniform( -1.0, 1.0 );
```

Comment:
```suggestion
		s[ i ] = uniform( EPS, 5.0 );
		t[ i ] = uniform( 0.0, 1.0 );
		mu[ i ] = uniform( -10.0, 10.0 );
```

Additionally, you need to import `EPS` at the top.